packages/fxa-payments-server/src/components/PlanDetails/index.tsx

```diff
@@ -134,119 +207,138 @@ export const PlanDetails = ({
                 ))}
               </ul>
 
-              <div className="plan-details-total">
-                {coupon && coupon.discountAmount && (
+              {loading ? (
+                <div className="loading">
+                  <LoadingSpinner />
+                </div>
+              ) : (
+                <div className="plan-details-total">
                   <div className="row-divider-grey-200 mb-4 pb-6">
-                    <div className="plan-details-total-inner">
-                      <Localized id="plan-details-list-price">
-                        <div>List Price</div>
-                      </Localized>
-
-                      <Localized
-                        id={`list-price`}
-                        attrs={{ title: true }}
-                        vars={{
-                          amount: getLocalizedCurrency(amount, currency),
-                          intervalCount: interval_count,
-                        }}
-                      >
-                        <div>
-                          {getLocalizedCurrencyString(amount, currency)}
-                        </div>
-                      </Localized>
-                    </div>
+                    {subTotal && (
+                      <div className="plan-details-total-inner">
+                        <Localized id="plan-details-list-price">
+                          <div>List Price</div>
+                        </Localized>
+                        <Localized
+                          id={`list-price`}
+                          attrs={{ title: true }}
+                          vars={{
+                            amount: getLocalizedCurrency(subTotal, currency),
+                            intervalCount: interval_count,
+                          }}
+                        >
+                          <div>
+                            {getLocalizedCurrencyString(subTotal, currency)}
+                          </div>
+                        </Localized>
+                      </div>
+                    )}
+                    {taxAmount && (
+                      <div className="plan-details-total-inner">
+                        <Localized id="plan-details-tax">
+                          <div>Taxes & fees</div>
+                        </Localized>
```

Comment:
`plan-details-tax` missing from `main.ftl`.

Reply:
added to ftl file